Fw/Com/ComPacket.hpp

```diff
@@ -24,6 +24,7 @@ namespace Fw {
                 FW_PACKET_LOG, // !< Log type - outgoing
                 FW_PACKET_FILE, // !< File type - incoming and outgoing
                 FW_PACKET_PACKETIZED_TLM, // !< Packetized telemetry packet type
+                FW_PACKET_DP, //!< Data product packet
```

Comment:
I'm coming into this fairly ignorant to the detailed F' data product architecture, but if data products are stored as files couldn't we just downlink them like standard files? Is there a benefit to giving DPs their own specific packet type?

Reply:
Yes, the plan is to downlink data products as standard files. The idea is that a data product is stored on disk as a header plus data. The header has a data product packet ID inside it, similarly to other F Prime data units such as com buffers. When a data product file is downlinked, it works like downlinking any file: the file is divided into packets, and each packet has a header with a file packet ID.

Reply:
In a review we discussed the fact that a data product packet is not intended to be used as a "com packet," i.e., as a packet in a transmission protocol. However, in current F Prime practice, "com packet" descriptors are not just for transmission; they also appear in data payloads, e.g., telemetry buffers and event buffers stored on disk. @LeStarch suggested that we move the ComPacket enumeration into a DataPacket enumeration. We can do this in a future PR.